contrib/powershell/Monitor-Room-Resources.ps1

```diff
@@ -0,0 +1,40 @@
+# Set the folder paths
```

Comment:
Add a big block comment at the top explaining what this script is for, and giving an example of how and where to run it.